src/libmongoc/tests/test-mongoc-uri.c

```diff
@@ -2772,6 +2752,7 @@ test_uri_install (TestSuite *suite)
    TestSuite_Add (suite, "/Uri/utf8", test_mongoc_uri_utf8);
    TestSuite_Add (suite, "/Uri/duplicates", test_mongoc_uri_duplicates);
    TestSuite_Add (suite, "/Uri/int_options", test_mongoc_uri_int_options);
+   TestSuite_Add (suite, "/Uri/bool_options", test_mongoc_uri_bool_options);
```

Comment:
After warning, can you add a test to check that a combination of timeoutMS and socketTimeoutMS / waitQueueTimeoutMS produces a warning?

Reply:
Done.

Reply:
Thanks!